server/webdriver_tests/timeline_test.py

```diff
@@ -89,6 +92,29 @@ def test_check_statvar_and_uncheck(self):
         # Assert there is one chart.
         self.assertEqual(len(charts), 1)
 
+    def test_place_search_box_and_remove_place(self):
+        """Test the timeline tool place search can work correctly."""
+        # Add place into place search box.
+        self.driver.get(self.url_ + TIMELINE_URL + STATVAR_URL_1)
+        time.sleep(3)
+        search_box = self.driver.find_element_by_id("search")
+        search_box_input = search_box.find_element_by_id("ac")
+        search_box_input.send_keys(PLACE_SEARCH)
+        time.sleep(2)
+        search_results = self.driver.find_elements_by_class_name("pac-item")
+        ca_result = search_results[0]
+        ca_result.click()
+        time.sleep(3)
+        charts = self.driver.find_elements_by_class_name("card")
+        self.assertEqual(len(charts), 1)
```

Comment:
can you assert on the <svg> content, eg, the svg has two lines now.

Reply:
Done